AV-103170: Document Query REST API passthrough operations (#386)
couchbaselabs/docs-devex: modules/n1ql/pages/n1ql-language-reference/createcollection.adoc

== Related Links

> * An overview of scopes and collections is provided in {scopes-and-collections}[].

Reviewer: ```suggestion
*  For an overview of scopes and collections, see {scopes-and-collections}[].
```

Author: Fixed

> * Step-by-step procedures for management are provided in {manage-scopes-and-collections}[].

Reviewer: ```suggestion
* For step-by-step procedures for management, see {manage-scopes-and-collections}[].
```

Author: Fixed

> * Refer to {scopes-and-collections-api}[Buckets, Scopes, and Collections] to manage scopes and collections with the Management API.

Reviewer: ```suggestion
* To manage scopes and collections with the Management API, see {scopes-and-collections-api}[Buckets, Scopes, and Collections].
```

Author: Fixed

> * Refer to the {couchbase-cli-collection-manage}[Couchbase Shell Documentation] to manage scopes and collections with the Couchbase Shell.

Reviewer: ```suggestion
* To manage scopes and collections with the Couchbase Shell, see {couchbase-cli-collection-manage}[Couchbase Shell Documentation].
```

Author: Fixed
